Automattic/jetpack
# Search: Remove require attribute on search inputs

Discovered while testing #20448.

#### Changes proposed in this Pull Request:
* Context: With WordPress 5.8, search widgets are automatically converted into search blocks which now output a search input with a newly added `required` attribute. This change prevents the user from spawning the Instant Search interface if the input is blank.
* This PR removes the `required` attribute when binding onto the search input.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* On a WordPress 5.8 installation with a Jetpack Search subscription, add a Search block to a page (or sidebar area).
* With Instant Search disabled, navigate to the page (or sidebar) with the Search block. Verify that the search input has the `required` attribute.
* Enable Instant Search via the Performance Settings page in wp-admin and repeat the above step. Ensure that the `required` attribute is no longer present.
